Misc/reputation-ipv6.json

```
@@ -0,0 +1,140 @@
+{
+    "id": "ipv6",
+    "version": -1,
+    "fromVersion": "5.0.0",
```

The `fromVersion` should be `5.5.0`, no?
```suggestion
    "fromVersion": "5.5.0",
```